repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/container/bindings.xjb

```diff
@@ -24,8 +24,10 @@
           version="2.0"
           xsi:schemaLocation="http://java.sun.com/xml/ns/jaxb http://java.sun.com/xml/ns/jaxb/bindingschema_2_0.xsd"
           schemaLocation="container-configuration.xsd">
-
     <schemaBindings>
         <package name="org.openrepose.core.container.config"/>
     </schemaBindings>
-</bindings>
\ No newline at end of file
+    <globalBindings>
```

Comment:
is this change needed, if so why?

Reply:
This ended up being needed by the `DeploymentConfigPatchUtil.deepCopy()` method so the objects are forced to implement `Serializable`.